apm-sniffer/apm-agent-core/src/main/java/org/skywalking/apm/agent/core/plugin/PluginCfg.java

```diff
@@ -1,34 +1,36 @@
 package org.skywalking.apm.agent.core.plugin;
 
-import org.skywalking.apm.util.StringUtil;
-
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.util.ArrayList;
 import java.util.List;
+import org.skywalking.apm.agent.core.conf.Config;
 
 public enum PluginCfg {
     INSTANCE;
 
-    private List<String> pluginClassList = new ArrayList<String>();
+    private List<PluginDefine> pluginClassList = new ArrayList<PluginDefine>();
 
     void load(InputStream input) throws IOException {
         try {
             BufferedReader reader = new BufferedReader(new InputStreamReader(input));
-            String pluginDefineClassName = null;
-            while ((pluginDefineClassName = reader.readLine()) != null) {
-                if (!StringUtil.isEmpty(pluginDefineClassName)) {
-                    pluginClassList.add(pluginDefineClassName.trim());
+            String pluginDefine = null;
+            while ((pluginDefine = reader.readLine()) != null) {
+                PluginDefine plugin = PluginDefine.build(pluginDefine);
+                if (!plugin.disabled(Config.Plugin.DISABLED_PLUGINS)) {
```

Comment:
You are using a static field as a parameter, why? You can use is in `disabled` method, no parameter is needed.

And I prefer the method name should be `boolean enable()`.